projects/ngrx.io/content/examples/store-elelments/.gitignore

```diff
@@ -0,0 +1,46 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
I haven't taken a full look but you made a typo for the folder name, `elelments` should be `elements`.

Reply:
Thanks for noticing it, I don't know why I keep messing with grammar :).